frontend/src/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -36,7 +37,7 @@ class CrossingListItem extends React.Component {
         durationId: (this.state.selectedStatus === statusConstants.LONGTERM ? this.state.selectedDuration : null),
         notes: this.state.notes
       },
-      refetchQueries: [{ query: crossingsQuery }]
+      refetchQueries: [{ query: crossingsQuery }, {query: statusCountsQuery}]
```

Comment:
@ifsimicoded questioned the performance of this refetch of the full list of crossings. When @briaguya tested it with many crossings, it was noticeably slower. We should dig deeper into how this query can be optimized, if Apollo has some suggested patterns, do we need to adopt pagination as a strategy to deal with this?

Google: "Apollo optimistic UI"
https://dev-blog.apollodata.com/tutorial-graphql-mutations-optimistic-ui-and-store-updates-f7b6b66bf0e2

Reply:
https://github.com/cityofaustin/ctxfloods/issues/62